piton/tools/src/proto/common/setup.tcl

```diff
@@ -112,34 +112,44 @@ for {set k 0} {$k < $::env(PITON_NUM_TILES)} {incr k} {
 
 puts "INFO: Using Defines: ${ALL_DEFAULT_VERILOG_MACROS}"
 
+# credit goes to https://github.com/PrincetonUniversity/openpiton/issues/50 
+# and https://www.xilinx.com/support/answers/72570.html
+set tmp_PYTHONPATH $env(PYTHONPATH)
+set tmp_PYTHONHOME $env(PYTHONHOME)
+unset env(PYTHONPATH)
+unset env(PYTHONHOME)
+
 # Pre-process PyHP files
 source $DV_ROOT/tools/src/proto/common/pyhp_preprocess.tcl
 set ALL_RTL_IMPL_FILES [pyhp_preprocess ${ALL_RTL_IMPL_FILES}]
 set ALL_INCLUDE_FILES [pyhp_preprocess ${ALL_INCLUDE_FILES}]
 
 
 if  {[info exists ::env(PITON_ARIANE)]} {
+<<<<<<< HEAD
   puts "INFO: compiling DTS and bootroms for Ariane (MAX_HARTS=$::env(PITON_NUM_TILES), UART_FREQ=$env(CONFIG_SYS_FREQ))..."
-  
-  
-  # credit goes to https://github.com/PrincetonUniversity/openpiton/issues/50 
-  # and https://www.xilinx.com/support/answers/72570.html
-  set tmp_PYTHONPATH $env(PYTHONPATH)                                                                               
-  set tmp_PYTHONHOME $env(PYTHONHOME)                                                                               
-  unset ::env(PYTHONPATH)                                                                                           
+=======
+  set tmp_PYTHONPATH $env(PYTHONPATH)
+  set tmp_PYTHONHOME $env(PYTHONHOME)
+  unset ::env(PYTHONPATH)
   unset ::env(PYTHONHOME)
-  
+>>>>>>> a13043d... add initial support for supporting uboot spl
```

Comment:
Minor thing but looks like you have a merge conflict comment in here

Reply:
Apologies, must have missed this. This is resolved now.